setup.py

```diff
@@ -162,16 +162,16 @@ def run(self, *args, **kw):
                     ext.cmake_install_prefix, ext.cmake_install_python_files
                 )
                 if os.path.isdir(src_py_dir):
-                    copy_tree(src_py_dir, self.build_lib)  # accepts existing dst dir
+                    copytree(src_py_dir, self.build_lib,dirs_exist_ok=True)  # accepts existing dst dir
```

Comment:
```suggestion
                    copytree(src_py_dir, self.build_lib, dirs_exist_ok=True)
```
Don't think that comment is needed any longer.